api/src/main/java/org/pmiops/workbench/api/util/SQLGenerator.java

```diff
@@ -4,14 +4,51 @@
 import com.google.cloud.bigquery.QueryRequest;
 import com.google.common.collect.ArrayListMultimap;
 import com.google.common.collect.ListMultimap;
+import org.pmiops.workbench.config.WorkbenchConfig;
 import org.pmiops.workbench.model.SearchParameter;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import javax.inject.Provider;
 import java.util.*;
 
 @Service
 public class SQLGenerator {
 
+    @Autowired
+    private Provider<WorkbenchConfig> workbenchConfig;
+
+    private static final String CRITERIA_QUERY =
+            "SELECT id,\n" +
+                    "type,\n" +
+                    "code,\n" +
+                    "name,\n" +
+                    "est_count,\n" +
+                    "is_group,\n" +
+                    "is_selectable,\n" +
+                    "domain_id\n" +
+                    "FROM `%s.%s.%s`\n" +
+                    "WHERE parent_id = @parentId\n" +
+                    "order by id asc";
+
+    /*
+     * The format placeholders stand for, in order:
+     *  - the table prefix (e.g., something like "pmi-drc-api-test.synpuf")
+     *  - the table name
+     *  - the table prefix again
+     *  - the *_SOURCE_CONCEPT_ID column identifier for that table
+     *  - the *_SOURCE_VALUE column identifier for that table
```

Comment:
Nit: this comment is no longer correct given that we changed the SQL to use `AND b.CONCEPT_CODE ...` rather than `AND a.%s ...`.

On that note: each line that reads `table.put("sourceValueColumn", "*_SOURCE_VALUE");` can probably go away.